docs/apm/advanced-queries.asciidoc

```diff
@@ -1,26 +1,18 @@
-[[query-bar]]
+[[advanced-queries]]
 === Advanced queries
 
-The query bar is a powerful data query feature.
-Similar to the query bar in {kibana-ref}/discover.html[Discover],
-it enables you to pass advanced queries on your data to filter on particular pieces of information that you're interested in.
-It comes with a handy autocomplete that helps find the fields and even provides suggestions to the data they include.
-You can select the query bar and hit the down arrow on your keyboard to begin seeing recommendations.
-
 When querying, you're simply searching and selecting data from fields in Elasticsearch documents.
 It may be helpful to view some of your documents in {kibana-ref}/discover.html[Discover] to better understand how APM data is stored in Elasticsearch. 
 
-The query bar is available in the Services, Transactions, Errors, Metrics, and Traces views,
-and any input will persist as you move between pages.
-
-TIP: Interactions with the query bar change the URL of the page you're on.
+Interactions with the query bar change the URL of the page you're on.
```

Comment:
May I suggest changing this to something along the lines of; "Queries entered into the query bar are also added as parameters to the URL, so it's easy to share a specific query or view with others." I was a little stumped on that "it changes the URL of the page you're on".

Reply:
Great stuff